ac/ac-train/src/index.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+
+import { type ActivityPackageT } from 'frog-utils';
+import ActivityRunner from './ActivityRunner';
+import dashboard from './Dashboard';
+
+const meta = {
+  name: 'Train Activity',
+  shortDesc: 'New activity, no description available',
+  description: 'New activity, no description available',
+  exampleData: [
+    {
+      title: 'Normal',
+      config: {
+        timeOfEachInstance: 30000
+      },
+      data: {}
+    },
+    {
+      title: 'Quick',
+      config: {
+        timeOfEachInstance: 15000
+      },
+      data: {}
+    }
+  ]
+};
+
+const config = {
+  type: 'object',
+  required: ['timeOfEachInstance'],
+  properties: {
```

Comment:
Could we specify here if it should be in seconds or milliseconds?

Reply:
^ @lfaucon how would I do this?